controllers/k8ssandra/k8ssandracluster_controller_test.go

```diff
@@ -51,31 +48,41 @@ import (
 const (
 	timeout  = time.Second * 5
 	interval = time.Millisecond * 500
+
+	k8sCtx0 = "cluster-0"
+	k8sCtx1 = "cluster-1"
+	k8sCtx2 = "cluster-2"
 )
 
 var (
-	defaultStorageClass = "default"
-	testEnv             *testutils.MultiClusterTestEnv
-	managementApi       = &fakeManagementApiFactory{}
+	defaultStorageClass  = "default"
+	testEnv              *testutils.MultiClusterTestEnv
+	managementApiFactory = &testutils.FakeManagementApiFactory{}
+
+	systemKeyspaces = []string{"system_auth", "system_distributed", "system_traces"}
 )
 
 func TestK8ssandraCluster(t *testing.T) {
 	ctx := testutils.TestSetup(t)
 	ctx, cancel := context.WithCancel(ctx)
-	testEnv = &testutils.MultiClusterTestEnv{}
+	testEnv = &testutils.MultiClusterTestEnv{
+		BeforeTest: func() {
+			managementApiFactory.Reset()
+		},
+	}
 
 	reconcilerConfig := config.InitConfig()
 
-	reconcilerConfig.DefaultDelay = 100 * time.Millisecond
-	reconcilerConfig.LongDelay = 300 * time.Millisecond
+	reconcilerConfig.DefaultDelay = 1 * time.Second
```

Comment:
Weren't these set to lower values originally to speed up the tests? Is there a reason to make them slower again?